app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersScreen.kt

```diff
@@ -151,20 +168,25 @@ private fun ReviewersScreenContent(
 
 @Composable
 private fun ReviewersList(
-    reviewers: List<Reviewer>,
+    data: Data,
+    pullRefreshState: PullRefreshState,
     modifier: Modifier,
     onNotifyClick: (ReviewersAction) -> Unit,
+    isLoading: Boolean,
 ) {
-    LazyColumn(
-        modifier = modifier.fillMaxWidth(),
-    ) {
-        itemsIndexed(reviewers) { index, reviewer ->
-            ReviewerItem(
-                reviewer = reviewer,
-                index = index,
-                onNotifyClick = onNotifyClick,
-            )
+    Box(modifier = Modifier.pullRefresh(pullRefreshState)) {
```

Comment:
suggestion: This Box with PullRefreshIndicator is done twice for two different screens, maybe it's worth considereing as common pattern and extracting it as a component?